opennlp-tools/src/main/java/opennlp/tools/sentdetect/lang/Factory.java

```diff
@@ -35,7 +35,7 @@
 
   public static final char[] thEosCharacters = new char[] { ' ','\n' };
 
-  public static final char[] jpEosCharacters = new char[] {'。', '！', '？'};
+  public static final char[] jaEosCharacters = new char[] {'。', '！', '？'};
```

Comment:
Would be better to use the three letter codes in the variable name. It was decided to drop two letter codes from OpenNLP.